packages/jaeger-ui/package.json

```diff
@@ -17,6 +17,8 @@
     "@babel/preset-typescript": "^7.21.0",
     "@svgr/babel-plugin-transform-svg-component": "^8.0.0",
     "@svgr/babel-preset": "^8.0.0",
+    "@testing-library/jest-dom": "^5.17.0",
```

Comment:
why is this explicitly added to deps? It's not used in the code, looks like it's a transitive dep that will be included in the lock file

Reply:
This would need to be installed when migrating from `mount` function (in Enzyme) to RTL, according to the[ official docs.](https://testing-library.com/docs/react-testing-library/migrate-from-enzyme/)